tests/testthat/test-regressions.R

```diff
@@ -131,16 +131,6 @@ test_that("summarize_row_groups at top level works", {
     expect_equal(dim(tbl), c(1, 1))
 })
 
-
-test_that("add_colcounts works as first call", {
```

Comment:
Keep this and move it to deprecated tests

Reply:
@gmbecker do you mean move it to the test-deprecated file? That's been done [here](https://github.com/Roche/rtables/pull/466/files#diff-da4f1b82827fd77584c3e10c13ad410b26e2d1b3793c70821ff78db809dc3b38R78).